client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -331,9 +331,13 @@
             templateUrl: 'scripts/superdesk-packaging/views/sd-package-items-edit.html',
             link: function(scope, elem, attrs, ngModel) {
                 scope.$on('package:addItems', function(event, data) {
-                    var groupIndex = _.findIndex(scope.list, {id: 'main'});
-                    for (var i = 0; i < data.length; i++) {
-                        scope.list[groupIndex].items.unshift(packages.getReferenceFor(data[i]));
+                    var groupIndex = _.findIndex(scope.list, {id: data.group});
+                    if (groupIndex === -1) {
+                        scope.list.push({id: data.group, items: []});
+                        groupIndex = scope.list.length - 1;
+                    }
+                    for (var i = 0; i < data.items.length; i++) {
+                        scope.list[groupIndex].items.unshift(packages.getReferenceFor(data.items[i]));
                     }
```

Comment:
That is correct, yes.
And just to make it clear, do performance discussions also fall under this, or should these be, generally speaking, discussed outside of PRs?

The rest does indeed not belong here, including using terms carrying a negative connotation (e.g. _nitpicking_) for what is a sincere attempt to make the code base better.

Over & out.